update flows samples to use v1 API (#2633)
knative/docs: docs/eventing/samples/sequence/sequence-reply-to-sequence/README.md

### Create the PingSource targeting the first Sequence

> This will create a PingSource which will send a CloudEvent with
> `{"message": "Hello world!"}` as the data payload every 2 minutes.

Reviewer: ```suggestion
This example creates a PingSource that sends a CloudEvent with
`{"message": "Hello world!"}` as the data payload every 2 minutes.
```
